webpack/assets/javascripts/react_app/components/common/Dropdown.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Dropdown = ({ links, id }) => {
+  const listLinks = links.map((link, i) => {
+    const liKey = `notification-link-${i}-${id}`;
+
+    return (
+      <li key={liKey} id={liKey}>
+        <a target="_blank" href={link.href}>{link.title}</a>
+      </li>
+    );
+  });
+
+  return (
+    <div className="dropdown pull-right dropdown-kebab-pf">
+      <button className="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight{id}"
+               data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+        <span className="fa fa-ellipsis-v"></span>
+      </button>
```

Comment:
Agree. We should try to not depend on bootstrap at all, especially on simple or trivial components. A bit of css wil get us where we need.